tests/unit/resources/__snapshots__/lists.test.ts.snap

```diff
@@ -64,68 +64,3 @@ exports[`lists .list() should retrieve the next page 1`] = `
   },
 ]
 `;
-
-exports[`lists .list() should retrieve the next page 2`] = `
```

Comment:
Could we double-check that this is intentional?

Reply:
Yes, this is indeed intentional, as there were previously two tests doing the same thing and they have been refactored to be one single test, rendering one of the snapshots obsolete.